### PR TITLE
Ref #37

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
Add calc_bins() to stats.py and add corresponding test routine.

### CODE REVIEW

File: src/emcpy/stats.py [hunk 1]
```diff
@@ -285,3 +285,28 @@ def bootstrap(insample, level=.95, estimator='mean', nrepl=10000):
...
+        eval_type: (str, default 'omf') options are 'omf', 'oma', 'observation', 'hofx'
+
+    Returns:
+        bins: (int) number of bins
+        binsize: (float) size of bins
+    """
+    binsize = (_np.max(data) - _np.min(data))/_np.sqrt(len(data))
+
+    if eval_type.lower() == 'omf' or eval_type.lower() == 'oma':
+        start = 0 - (4 * _np.std(data))
```
Comment (kevindougherty-noaa) [inline src/emcpy/stats.py]: Do you think we should make the number of standard deviations an option? We can set the default to `nstdevs=4`. @CoryMartin-NOAA what do you think?

Comment (CoryMartin-NOAA) [reply src/emcpy/stats.py]: Yeah probably should be an option. 4 standard deviations covers a lot/most of the distribution but I bet 3 is probably good enough for many applications.

**Review by CoryMartin-NOAA — COMMENTED** [review]
Looks good, but I agree with @kevindougherty-noaa , having options never hurts.

File: src/emcpy/stats.py (hunk 1 above)
Comment (EdwardSafford-NOAA) [reply src/emcpy/stats.py]: Ok, I'll make that change.  Give me an hour or so; I'm in the middle of something else now.

Comment (EdwardSafford-NOAA) [reply src/emcpy/stats.py]: I added an nstd argument which defaults to 4.

**Review by CoryMartin-NOAA — APPROVED** [review]

**Pull request merged**